### ISSUE TITLE
ExtJS v6

### ISSUE THREAD

**Issue by niallobrien** [burnnat/eslint-plugin-extjs#5]
Hi - any plans to update this for ExtJS v6.x? Thanks in advance.

**Comment by burnnat** [comment]
Sorry, I'm rarely working with ExtJS these days so I don't know enough about the changes in 5.x and 6.x to update it myself. I am happy to accept pull requests though, if you or someone else is so inclined!

**Comment by niallobrien** [comment]
Hey @burnnat - I'd be happy to contribute - just wondering if you ever ran ESlint in conjunction with Sencha Cmd or did you just run it in a separate process? Thanks in advance.

**Comment by burnnat** [comment]
No, I always ran ESLint separately, either directly through the command or via editor/IDE integration.

**Comment by ericpoe** [comment]
A nice thread between @niallobrien and Mitchell Simoens on a way to incorporate ESLint into the build system. https://twitter.com/LikelyMitch/status/900009407685115904

I haven't tried it out yet as I'm just now looking into my options for ESLint in ExtJS 4 and 6.
